ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/config/WorkflowConfigParserSpec.scala

```diff
@@ -141,20 +146,41 @@ class WorkflowConfigParserSpec extends AnyWordSpec with Matchers {
       )
     }
 
-    "parse no submission configuration" in {
+    "parse empty submission configuration" in {
       val yaml =
         """submission:
           |  type: empty
         """.stripMargin
+      parseYaml(yaml) shouldBe Right(
+        WorkflowConfig(
+          submission = Some(WorkflowConfig.EmptySubmissionConfig),
+          streams = Nil,
+        )
+      )
+    }
+
+    "parse fibonacci submission configuration" in {
+      val yaml =
+        """submission:
+          |  type: fibonacci
+          |  num_instances: 500
+          |  unique_parties: true
+          |  value: 7
+        """.stripMargin
 
       parseYaml(yaml) shouldBe Right(
         WorkflowConfig(
           submission = Some(
-            WorkflowConfig.EmptySubmissionConfig
```

Comment:
I think we should still be able to support an empty configuration, no? maybe adding instead of replacing the test would be more preferred.

Reply:
The EmptySubmission was only really introduced to help with testing.  Having `submission = None` is how a no submission stage configuration is represented.